java/ql/lib/semmle/code/java/frameworks/apache/Collections.qll

```diff
@@ -182,6 +204,695 @@ private class ApacheKeyValueModel extends SummaryModelCsv {
+          ".list;AbstractLinkedList;true;removeFirst;;;Element of Argument[-1];ReturnValue;value",
+          ".list;AbstractLinkedList;true;removeLast;;;Element of Argument[-1];ReturnValue;value",
+          ".list;AbstractLinkedList;true;updateNode;;;Argument[1];Element of Argument[-1];value",
+          ".list;AbstractLinkedList;true;updateNode;;;Argument[1];Element of Argument[0];value",
+          ".list;AbstractLinkedList;true;createNode;;;Argument[0];Element of ReturnValue;value",
+          ".list;AbstractLinkedList;true;addNodeBefore;;;Argument[1];Element of Argument[-1];value",
+          ".list;AbstractLinkedList;true;addNodeBefore;;;Argument[1];Element of Argument[0];value",
+          ".list;AbstractLinkedList;true;addNodeAfter;;;Argument[1];Element of Argument[-1];value",
+          ".list;AbstractLinkedList;true;addNodeAfter;;;Argument[1];Element of Argument[0];value",
+          ".list;AbstractLinkedList;true;addNode;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;AbstractLinkedList;true;addNode;;;Element of Argument[0];Element of Argument[1];value",
+          ".list;AbstractLinkedList;true;getNode;;;Element of Argument[-1];Element of ReturnValue;value",
+          ".list;AbstractLinkedList;true;createSubListIterator;;;Element of Argument[-1];Element of ReturnValue;value",
+          ".list;AbstractLinkedList;true;createSubListIterator;;;Element of Argument[0];Element of ReturnValue;value",
+          ".list;AbstractLinkedList;true;createSubListListIterator;;;Element of Argument[-1];Element of ReturnValue;value",
+          ".list;AbstractLinkedList;true;createSubListListIterator;;;Element of Argument[0];Element of ReturnValue;value",
+          ".list;AbstractLinkedList$LinkedListIterator;true;LinkedListIterator;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;AbstractLinkedList$LinkedListIterator;true;getLastNodeReturned;;;Element of Argument[-1];Element of ReturnValue;value",
+          ".list;AbstractLinkedList$LinkedSubList;true;LinkedSubList;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;AbstractLinkedList$LinkedSubListIterator;true;LinkedSubListIterator;;;Element of Argument[0];Element of Argument[-1];value",
+          ".list;AbstractLinkedList$Node;true;Node;(Object);;Argument[0];Element of Argument[-1];value",
+          ".list;AbstractLinkedList$Node;true;Node;(Node,Node,Object);;Element of Argument[0];Element of Argument[-1];value",
+          ".list;AbstractLinkedList$Node;true;Node;(Node,Node,Object);;Element of Argument[1];Element of Argument[-1];value",
+          ".list;AbstractLinkedList$Node;true;Node;(Node,Node,Object);;Argument[2];Element of Argument[-1];value",
+          ".list;AbstractLinkedList$Node;true;Node;(Node,Node,Object);;Argument[2];Element of Argument[0];value",
+          ".list;AbstractLinkedList$Node;true;Node;(Node,Node,Object);;Argument[2];Element of Argument[1];value",
```

Comment:
Same as above, not sure how `value` affects prev and next nodes here

Reply:
See :fish: